CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ add_library(${PROJECT_NAME}
         Tinyxml2Serializer.cpp
         Tinyxml2Deserializer.cpp
         Deserializer.cpp
-)
+ "Serializer.cpp" "Serializer.h")
```

Comment:
Issue: Seems like visual studio added these with its own formatting

Reply:
I fixed this before... Damn you, VS. I fixed it again. Good call.